optout/src/main/java/gov/cms/ab2d/optout/gateway/S3Gateway.java

```diff
@@ -0,0 +1,9 @@
+package gov.cms.ab2d.optout.gateway;
+
+import java.io.InputStreamReader;
+
+public interface S3Gateway {
+
+    InputStreamReader getS3Object();
```

Comment:
Let's use a method name that more clearly communicates its purpose (`getS3Object` might be too generic): `getOptOutFile`.

Reply:
done